types/node/vm.d.ts

```diff
@@ -64,17 +64,19 @@ declare module "vm" {
         /** @deprecated in favor of `script.createCachedData()` */
         produceCachedData?: boolean | undefined;
         /**
-         * Called during evaluation of this module when `import()` is called.
-         * If this option is not specified, calls to `import()` will reject with `ERR_VM_DYNAMIC_IMPORT_CALLBACK_MISSING`.
+         * Used to specify how the modules should be loaded during the evaluation of this script when `import()` is called. This option is
+         * part of the experimental modules API. We do not recommend using it in a production environment. For detailed information, see
+         * [Support of dynamic `import()` in compilation APIs](https://nodejs.org/api/vm.html#support-of-dynamic-import-in-compilation-apis).
```

Comment:
Some of these updated doc links go to latest (v21) instead of v20. Would be nice to align all of them